spec/pushmi_pullyu/aip/creator_spec.rb

```diff
@@ -2,25 +2,15 @@
 
 RSpec.describe PushmiPullyu::AIP::Creator do
   let(:workdir) { 'tmp/creator_spec' }
-  let(:options) do
-    { workdir: workdir,
-      fedora: { url: 'http://www.example.com:8983/fedora/rest',
```

Comment:
Only create variables for what your using in your tests. Creator class shouldn't need fedora/solr information.